cumulusci/core/config/tests/test_config_expensive.py

```diff
@@ -87,19 +86,19 @@ def test_load_universal_config_with_local(self, mock_class):
             expected_config = yaml.safe_load(f_expected_config)
         expected_config["tasks"]["newtesttask"] = {}
         expected_config["tasks"]["newtesttask"]["description"] = "test description"
-        self.assertEqual(config.config, expected_config)
+        assert config.config == expected_config
 
 
 @mock.patch("pathlib.Path.home")
```

Comment:
When is a patch decorator on a class in effect? Should this be replaced with a fixture?

Reply:
It's the same as patching each method individually. Seems to work.